[GH-1133] $network modifier description updated (#261)
AdguardTeam/KnowledgeBase: docs/general/ad-filtering/create-own-filters.md

We recommend to get acquainted with this [article](#regexp-support) for better understanding of regular expressions.

> `$network` modifier can only be included in rules alongside with `$app` and `$important` modifiers and no other ones.

Reviewer: в других местах как-то так выделяется
```suggestion
:::caution Restrictions

`$network` modifier can only be included in rules alongside with `$app` and `$important` modifiers and no other ones.

:::
```

Reviewer 2: ещё можно немного переформулировать:
```suggestion
The `$network` modifier can only be used in rules together with the `$app` and `$important` modifiers, and not with any other modifiers.
```
